### PR TITLE
[Components with more than 1 service] LPS-192202 Refactor export-import components

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-192202

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:sf

**Review by rafaprax — CHANGES_REQUESTED** [review]
Hey @MrJohn1911, why we need to add a breaking change note in the first commit?

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
Hey @MrJohn1911, about this https://github.com/rafaprax/liferay-portal/pull/1374/commits/d4e829736e17b8e5a38d15c8b6372ed921424cfd and this https://github.com/rafaprax/liferay-portal/pull/1374/commits/68f608bac563e5908dcb3dc406e896cf3615c3ef. It would be better if we:

1. Inline the  `ExportImportControllerRegistryUtil` into the `ExportImportManager`;
2. Replace the ServiceTracker usages by single `@Reference` references
3. Remove the unused interfaces

Thanks

File: portal-impl/src/com/liferay/portlet/exportimport/service/impl/ExportImportLocalServiceImpl.java [hunk 1]
```diff
@@ -1,779 +0,0 @@
-/**
- * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
- * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
- */
-
-package com.liferay.portlet.exportimport.service.impl;
-
```
Comment (rafaprax) [inline portal-impl/src/com/liferay/portlet/exportimport/service/impl/ExportImportLocalServiceImpl.java:7]: Hey @MrJohn1911, update the commit message to say that you moved the Service and ServiceImpl to the Manager class. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
ci:test:sf

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
forwarded to https://github.com/liferay-core-infra/liferay-portal/pull/6331

**Pull request closed without merging**